include/xtensor/xaxis_iterator.hpp

```diff
@@ -117,31 +117,30 @@ namespace xt
             auto nxt_strides = std::copy(e_strides.cbegin(), e_strides.cbegin() + axis, strides.begin());
             std::copy(e_strides.cbegin() + axis + 1, e_strides.end(), nxt_strides);
 
-            return strided_view(std::forward<CT>(e), std::move(shape),
-                std::move(strides), offset, e.layout());
+            return strided_view(
+                std::forward<CT>(e), std::move(shape), std::move(strides), offset, e.layout()
+            );
         }
     }
 
     template <class CT>
     template <class T, class CTA>
-    inline std::enable_if_t<std::is_pointer<T>::value, T>
-    xaxis_iterator<CT>::get_storage_init(CTA&& e) const
+    inline std::enable_if_t<std::is_pointer<T>::value, T> xaxis_iterator<CT>::get_storage_init(CTA&& e) const
     {
         return &e;
     }
 
     template <class CT>
     template <class T, class CTA>
-    inline std::enable_if_t<!std::is_pointer<T>::value, T>
-    xaxis_iterator<CT>::get_storage_init(CTA&& e) const
+    inline std::enable_if_t<!std::is_pointer<T>::value, T> xaxis_iterator<CT>::get_storage_init(CTA&& e) const
     {
         return e;
     }
 
     /**
      * @name Constructors
      */
-     //@{
+    //@{
```

Comment:
```suggestion
```
We should fix this manually too

Reply:
Why would we want to get rid of these comments? I think they are used by doxygen.

Reply:
Ah sorry, that I overlooked. What a horrible syntax ;)